docs/reference/search/search-your-data/run-knn-search.asciidoc

```diff
@@ -0,0 +1,248 @@
+[[run-knn-search]]
+== Run a k-nearest neighbor (kNN) search
```

Comment:
May be a better title would a word "Run" and just "k-nearest neighbor (kNN) search"?

Reply:
Thanks for the suggestion. I did this in b07f38b4176. I had to swap around some anchors with the kNN search API docs.